Package.swift

```diff
@@ -756,6 +766,7 @@ let package = Package(
         "FirebaseFirestoreSwift",
         "FirebaseFunctions",
         "FirebaseInAppMessaging",
+        "FirebaseInAppMessagingSwift",
```

Comment:
You may need to do something like `AppDistribution` and add in a condition. Also, can you add a line to make sure this compiles in the swift-test as well?

Reply:
First part fixed in fae7aa2.

@ryanwilson I'm unclear about what you mean by the line to ensure it compiles in swift-test.

Reply:
Added tests in c5fcc43.